### PR TITLE
improve the volpage results IDEAS

### PR DESCRIPTION
**Pull request by thesunshade**
NOTE: I have accidentally included adding `client/localization/elements/pali-tipitaka_en.json` into this PR. I can't figure out how to remove it. It is correct, but it really doesn't belong in this PR.

NOTE 2: This is really just a mockup. I doubt if the code is very stable in terms of dealing with errors. It may all need to be re-written.

In this PR I'm doing two things

1. improving the layout of the `volpage:` table so that the final row with all of the refs will wrap on mobile
2. transforming the raw refs into PTS citations (sort of).

Here is what it looks like now:
image

And here it is with my changes:
image

### CODE REVIEW

File: client/elements/sc-page-search.js [hunk 1]
```diff
@@ -462,6 +462,18 @@ class SCPageSearch extends LitLocalized(LitElement) {
       priorityAuthor || 'ms'
     }?${linkParamPart}`;
 
+    function formatRef (ref){
+      const romanNumerals = ["i", "ii", "iii", "iv", "v", "vi", "vii", "viii"];
+      let volpage = ref.replace(/pts-vp-pli/,"")
+      let [vol, page] = volpage.split(".")
+      return `${romanNumerals[vol-1]} ${page}`
+    }
```
[the remark this answers is not in the document]
Comment (ihongda) [reply client/elements/sc-page-search.js]: Bhante 🙏🏼 ,  Based on AI suggestions, we can modify the code to something like:

```
const formatRef = function(ref) {
  const romanNumerals = ["i", "ii", "iii", "iv", "v", "vi", "vii", "viii"];
  const volpage = ref.replace(/pts-vp-pli/,"");
  const [vol, page] = volpage.split(".");
  return `${romanNumerals[vol-1]} ${page}`;
};

const getBook = function(acronym) {
  const [book, number] = acronym.split(" ");
  return book;
};
```

**Comment by ihongda** [comment]
@thesunshade Bhante,  I took the liberty of making some modifications in your working branch.


screenshot_2024-03-06_01-40-22

**Comment by thesunshade** [comment]
> I took the liberty

Oh, this looks so much better! You always have **_total permission_** to modify any and all of the code I ever submit. Please don't be shy!

**Comment by thesunshade** [comment]
I really love your improvement. It brings these results in line with the regular result page format.

I notice that the sutta titles are in bold. That is different from the regular results.

You might want to put the results in some kind of div so that they weren't full width, and so they were centered on the screen. 

image

Also I notice that in the above screenshot the search bar extends across the whole screen, whereas in a normal search it does not:
image

**Comment by ihongda** [comment]
> Also I notice that in the above screenshot the search bar extends across the whole screen, whereas in a normal search it does not:

Normal search results are divided into two columns, while volpage search results only have one column, so it will automatically extend.

**Comment by thesunshade** [comment]
I'm just wondering if we could set a `max-width` so the search button wasn't so far away from the text input.

**Comment by ihongda** [comment]
>I notice that the sutta titles are in bold. That is different from the regular results.

fixed!

**Comment by ihongda** [comment]
> I'm just wondering if we could set a `max-width` so the search button wasn't so far away from the text input.

It should be possible.
Done.

**Review by ihongda — APPROVED** [review]

**Pull request merged**